net/proxy_resolution/pac_file_fetcher_impl_unittest.cc

```diff
@@ -171,7 +171,7 @@ TEST_F(PacFileFetcherImplTest, HttpMimeType) {
                                     TRAFFIC_ANNOTATION_FOR_TESTS);
     EXPECT_THAT(result, IsError(ERR_IO_PENDING));
     EXPECT_THAT(callback.WaitForResult(), IsOk());
-    EXPECT_EQ(u"-pac.txt-\n", text);
```

Comment:
Are you sure that it's OK that when we run it, the newline is dropped?

Reply:
Specifically, the file being fetched 'pac.txt' does have the newline, and the upstream test has always assumed that the newline is also read, so I don't think the test was wrong.